proto/config/vm.proto

```diff
@@ -94,4 +94,7 @@ message VmConfig {
   bool enforce_network_interface_order = 24;
   // The app should receive the Windows license embedded in ACPI tables
   bool enable_oem_win_license_key = 25;
+  // Disable vTPM for the VM, this defaults to false according to the
```

Comment:
Are you planning to make it configurable for an application at runtime? I mean, if you put it here in `vm.proto`, it will become part of the `FixedResources` in `AppInstanceConfig`, which means the user would need to update the app template and perform a Purge & Update to change this setting. That might be the right approach for your use case.

But if you want it to be more dynamic - allowing changes without a purge - then it should go to https://github.com/lf-edge/eve-api/blob/f2951f18534ec5c4b218104d5279575dae03de0a/proto/config/appconfig.proto#L106

Reply:
As @eriknordmark has already approved, I guess you are doing it right. Just want to be sure)

Reply:
Makes sense!